omniscidb/IR/OpType.h

```diff
@@ -250,3 +248,5 @@ inline std::string toString(hdk::ir::WindowFunctionKind kind) {
 inline std::ostream& operator<<(std::ostream& os, hdk::ir::WindowFunctionKind kind) {
   return os << toString(kind);
 }
+
+}  // namespace hdk::ir
```

Comment:
It was intentional to have `toString` and `operator<<` in the global namespace because we want to reload the global `toString` method and allow stream values without `using namespace hdk::ir`. We have similar methods for other enums in `hdk::ir::` namespace.

Reply:
The problem w/ that seems to be in the logger `CHECK_EQ` macro, which is trying to access the `operator<<` overload and can't find it.

Reply:
I'm surprised that moving it from the global namespace to `hdk::ir` helps to find it. Maybe we have some ambiguous overloads?